examples/jupyter-docker-stacks-julia-image/Dockerfile

```diff
@@ -1 +1,13 @@
-FROM jupyter/datascience-notebook:julia-1.4.1
\ No newline at end of file
+FROM jupyter/datascience-notebook:julia-1.5.2
+
+# Install the AWS CLI:
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip
+USER root
+RUN ./aws/install
+USER $NB_UID
+
+# Install various Python utilities for SageMaker, and PyCall to enable using them from Julia:
+RUN julia -e 'import Pkg; Pkg.update()' && \
+    julia -e 'using Pkg; pkg"add PyCall"; pkg"precompile"' && \
+    pip install boto3 sagemaker sagemaker-experiments sagemaker-studio-image-build smdebug
```

Comment:
Could we pin this to range of the current major version and put each package on a new line for readability? For e.g.

```
pip install \
        'boto3>1.0<2.0'       \
        'sagemaker<3'

```

Reply:
Thanks for the feedback - done now in the commits added below! I pinned the Julia package also, and figured we should pin minor version of any 0.x packages, since in standard semver these would be less stable?

Reply:
> figured we should pin minor version of any 0.x packages, since in standard semver these would be less stable?

Yep, that is a reasonable place to start.